packages/next/src/index.ts

```diff
@@ -1142,6 +1142,11 @@ export const build: BuildV2 = async ({
       appPathRoutesManifest,
     });
 
+    /**
+     * Originally, you had to have a `pages/api` route in order to add a preview
```

Comment:
nit, but I think maybe this comment could be summed up as preview mode is a pages router concept, draft mode is an app router concept.